io/include/pcl/io/impl/pcd_io.hpp

```diff
@@ -197,6 +197,11 @@ pcl::PCDWriter::writeBinary (const std::string &file_name,
   // Prepare the map
 #if _WIN32
   HANDLE fm = CreateFileMappingA (h_native_file, NULL, PAGE_READWRITE, 0, (DWORD) (data_idx + data_size), NULL);
+  if (fm == NULL)
+  {
+      throw pcl::IOException("[pcl::PCDWriter::writeBinary] Error during write ()!");
```

Comment:
Modify the error message to indicate that there was an error during  memory map creation.